travis/build_new_nightly

```diff
@@ -14,7 +14,8 @@ if [ "${TARGET_PLATFORM:-}" = "" ]; then
     exit $badusage
 fi
 
-PLATFORM_TYPE="${TARGET_PLATFORM%%/*}"
+PLATFORM_TYPE="$(echo "$TARGET_PLATFORM" | cut -d "/" -f1)"
+PLATFORM_VERSION="$(echo "$TARGET_PLATFORM" | cut -d "/" -f2 | cut -d "/" -f1)"
```

Comment:
My bash scripting experience is rather limited, so please bear with me.

1. Why do you not split the string using `/` as a delimiter?
2. IIRC `TARGET_PLATFORM` variables contain a single `/` character. The second cut operation would not actually do anything, right?

Reply:
1 - doing so actually
2 - true, I will remove the second one

Reply:
```bash
IFS=/; read -a TARGET <<<"$TARGET_PLATFORM"
PLATFORM_TYPE = $TARGET[0]
PLATFORM_VERSION = $TARGET[1]
```

Reply:
https://github.com/citusdata/tools/pull/83/commits/6b687fbc521536a9c4e51558a1f21933627b4f44